src/core/functions/table/match.cpp

```diff
@@ -33,6 +35,55 @@ namespace duckpgq {
 
 namespace core {
 
+namespace {
+
+// Get fully-qualified column names for the property graph [tbl], and insert
+// into set [col_names].
+void PopulateFullyQualifiedColName(
+    const vector<shared_ptr<PropertyGraphTable>> &tbls,
+    const case_insensitive_map_t<vector<string>> &tbl_name_to_aliases,
+    case_insensitive_set_t &col_names) {
+  for (const auto &cur_tbl : tbls) {
+    for (const auto &cur_col : cur_tbl->column_names) {
+      // It's legal to query by `<col>` instead of `<table>.<col>`.
+      col_names.insert(cur_col);
```

Comment:
I'm wondering if this will also be sufficient for property graphs on attached databases. Then we also need to add the catalog and schema (`"catalog"."schema"."table"."column"`). 
I would like to see a test case handle this. For examples of attached databases, see https://github.com/cwida/duckpgq-extension/blob/9016214bbbbfd4e0f0fd5f4d9dfa74c61ab9d385/test/sql/create_pg/attach_pg.test

Reply:
I think that's a good point, and it's not hard to get the catalog and schema name since they're already part of `PropertyGraphTable`.

However after some trials, I found it's failing due to another bind error:
https://github.com/cwida/duckdb-pgq/blob/4d5ca72d02b0844dc0acc6e605db5100b6761a29/src/planner/bind_context.cpp#L240-L255

Checking the variable, only `<table>.<column>` is stored in the `bindings` here.

Reply:
Alright, maybe we don't need the catalog and schema in that case. However, could you still add a test case to the attach_pg.test just to be sure?

Reply:
That makes sense; I added some more sql failure tests with catalog and schema prefix. Let me know if I understand correctly!